Update direct-routing-plan.md (#3671)
MicrosoftDocs/OfficeDocs-SkypeForBusiness: Teams/direct-routing-plan.md

For more information about Phone System licensing, see [Microsoft Teams add-on licensing](teams-add-on-licensing/microsoft-teams-add-on-licensing.md). 

> ## Supported end points 

Reviewer: ```suggestion
## Supported endpoints 
```

> You can use as end point:

Reviewer: ```suggestion
The following can be used as endpoints:
```

> - Any Teams cleintg. 

Reviewer: ```suggestion
- Any Teams client. 
```

> - Common Area Phones, plese see [Set up the Common Area Phone license for Microsoft Teams](https://docs.microsoft.com/en-us/microsoftteams/set-up-common-area-phones). Note you do not need Calling Plan license when setting the Common Area Phone with Direct Routing.

Reviewer: ```suggestion
- Common Area Phones. Please see [Set up the Common Area Phone license for Microsoft Teams](https://docs.microsoft.com/en-us/microsoftteams/set-up-common-area-phones) for more information. Note that you do not need a Calling Plan license when setting up a Common Area Phone with Direct Routing.
```

> - Skype for Business 3PIP phones. Please read [Skype for Business phones (3PIP) support with Microsoft Teams](https://techcommunity.microsoft.com/t5/Microsoft-Teams-Blog/Skype-for-Business-phones-3PIP-support-with-Microsoft-Teams/ba-p/789351)

Reviewer: ```suggestion
- Skype for Business 3PIP phones. Please read [Skype for Business phones (3PIP) support with Microsoft Teams](https://techcommunity.microsoft.com/t5/Microsoft-Teams-Blog/Skype-for-Business-phones-3PIP-support-with-Microsoft-Teams/ba-p/789351).
```
